.github/actions/update_versions_json/update_versions.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```
Can we have the full script as a nix script instead of an action? (/packages/scripts.nix)

Reply:
Should not be a problem, I'll rewrite it accordingly.